data/table.txt

```diff
@@ -8691,15 +8761,18 @@
 唩 唩 1 0 0 0 1 0 0 0 0 rhdv rhdv NA 0
 唪 唪 1 1 0 0 1 0 0 0 0 rqkq rqkq NA 11592
 唫 唫 1 0 1 0 1 0 0 0 0 rc rc,xxrc NA 0
-唬 唬 1 1 0 0 1 0 0 0 0 ryphn,rypu rypn,rypu NA 19994
+唬 唬 1 1 0 0 1 0 0 0 0 ryphn rypn NA 19994
+唬 唬 1 1 0 0 1 0 0 0 0 rypu rypu NA 0
```

Comment:
@yookoala [said](https://github.com/Cangjians/libcangjie/commit/d9d2e686ce6776ec67e89b4256dce42359597d84#r29099713):

> "rypn" returns nothing on MacOS. "rypu" is the standard mapping.

---

Should we remove `rypn` before running the script, then?

Reply:
So Wan Leung just said that Mac OS X uses the short-lived Cangjie version 4, not version 5. :open_mouth:

Discussing this with @yookoala, we decided to ignore all those comments about Mac.